Assets/Scripts/NPC/WalkingState.cs

```diff
@@ -46,7 +50,7 @@ protected override void FixedUpdateState(SmallEnemiesStateMachine enemy)
     private void MoveToWaypoint()
     {
         var moveDirection = currentWaypoint.transform.position - transform.position;
-        _moveVelocity = moveDirection.normalized * movingSpeed * Time.deltaTime;
+        _moveVelocity = moveDirection.normalized * (movingSpeed * Time.deltaTime);
```

Comment:
```suggestion
        _moveVelocity = moveDirection.normalized * movingSpeed * Time.deltaTime;
```
Deze haakjes zijn onnodig, je doet alles toch keer elkaar. Iets voorrang geven maakt niet uit.

Reply:
Rider maakte er een opmerking van `¯\_(ツ)_/¯`
![image](https://github.com/Team-Swamp/Brackeys-Gamejam-2023.2/assets/71002222/52d3bbea-95ff-42ea-a163-ba43c809aa20)

Reply:
In dat geval vind ik dat rider oneens heb, maakt niet uit welke volgworden het is, komt op het zelfde getal neer.
Maar als je het zo laat zoals het is vind ik het prima.

Reply:
@bas-boop https://stackoverflow.com/questions/57933831/unity-rider-order-of-multiplication-operations-is-inefficient

Ik denk wel dat het uitmaakt op basis van wat hier gezegd wordt. Is eigenlijk wel logisch

Reply:
Dat is intresant, zou dan net zoals het voorbeeld opschrijven zodat die haakjes weg kunnen.

Reply:
Zoals welk voorbeeld?

```1. scale=speed*time
2. X*=scale
3. Y*=scale
4. Z*=scale
```
Zoals dit? Lijkt me niet toch? Alleen maar zodat die haakjes weg kunnen?